### ISSUE TITLE
React.createClass VS extends React.Component

### ISSUE THREAD

**Issue by cychiuae** [oursky/philosophy#14]
`React.createClass` VS `extends React.Component`

Which one do you prefer?

**Comment by louischan-oursky** [comment]
[React.PureComponent](https://facebook.github.io/react/docs/react-api.html#react.purecomponent)

**Comment by akiroz** [comment]
doesn't it depend if you're using ES5 or ES6?

**Comment by cychiuae** [comment]
@akiroz you can use babel to do transformation so that is not a concern

**Comment by chankiyu22** [comment]
They are different, not just the coding style.
[https://toddmotto.com/react-create-class-versus-component/](https://toddmotto.com/react-create-class-versus-component/)

**Comment by akiroz** [comment]
@cychiuae yes, both with work in ES6 but the class approach seems more idiomatic in the ES6 code bases I've seen....

**Comment by cychiuae** [comment]
@chankiyu22 So, which one do you prefer?
